### PR TITLE
Fix Whitehall (extra dash) example

### PR DESCRIPTION
**Pull request by khaliqgant**
In addition:
- send back the actual station name and not the requested station name
  in case there is a mismatch
- log responses for local dev

### CODE REVIEW

**Comment by willwashburn** [comment]
:+1:

File: index.js [hunk 1]
```diff
@@ -13,23 +13,30 @@ exports.handler = function (event, context) {
...
             // return how many bikes and docks there are
             var say =
                 "There are " + bikes + " bikes " +
                 " and " + docks + " docks " +
-                " at " + station
+                " at " + name;
 
             context.succeed(response(say))
+            console.log(say);
+            context.succeed(response(say));
```
Comment (willwashburn) [inline index.js:28]: No need to call context.succeed() twice here, right?

Comment (willwashburn) [inline index.js:28]: @khaliqgant  ^^

**Comment by khaliqgant** [comment]
Nope, updated with above.

**Pull request merged**